docs/config-file-package-configuration-yml.md

```diff
@@ -1,38 +1,43 @@
 
-## Specifying the target package and provenance
+### When To Use
+Use a package configuration file to:
+
+-  mark files and directories as not included in released artifacts -- use it to make clear that license findings in documentation or tests in a package sources do not apply to the release (binary) artifact which is a dependency in your project
+-  overwrite scanner findings to correct identified license in a dependency for a specific file or directory
+
+# Package Configuration File Basics
+
+Each package configuration applies exactly to one *package id* and *provenance* which must be specified. The *provenance*
+can be specified as either a *source artifact* or a *VCS location and revision*. 
+
+Here is an example of a package configuration for `ansi-styles 4.2.1`, when the source artifact is (to be) scanned:
+
+```yaml
+  id: "NPM::ansi-styles:4.2.1"
+  source_artifact_url: "https://registry.npmjs.org/ansi-styles/-/ansi-styles-4.2.1.tgz"
+```
+
+If the source repository is (to be) scanned, then use the package configuration below:
 
-Each package configuration applies exactly to one package-Id and provenance which must be specified. The provenance
-can either specify a *source artifact* or a *VCS location and revision*. Thus in case one scans the *source artifact*
-and the *VCS* for each package, then two package configurations would be required in order to address issues with both
-scans. For example configurations for `ansi-styles 4.2.1` would specify the following... 
-  
-* to configure the VCS scan:
 ```yaml
   id: "NPM::ansi-styles:4.2.1"
   vcs:
     type: "Git"
     url: "https://github.com/chalk/ansi-styles.git"
     revision: "74d421cf32342ac6ec7b507bd903a9e1105f74d7"
 ```
-* to configure the source artifact scan:
-```yaml
-  id: "Maven:org.ossreviewtoolkit.ort:1.2.3"
-  source_artifact_url: "https://registry.npmjs.org/ansi-styles/-/ansi-styles-3.2.1.tgz"
-```
 
-## Specifying path excludes and license finding curations
+## Defining Path Excludes and License Finding Curations
 
-Path excludes are used to tell ORT that license findings in certain files can be ignored, for example because they
-belong to the documentation of the package. License finding curations are used to fix wrong scan results, for example
-if a wrong license was detected, or if a finding is a false positive. The entries for path excludes and license finding
-curations have the same syntax and semantic as the analog entries for the `ort.yml`, see
-[Excluding paths](config-file-ort-yml.md#excluding-paths) and
-[Curating license findings](config-file-ort-yml.md#curating-license-findings).
+Path excludes define which code is not part of the distributed release artifacts for a package, for example code found in the source repository but only used for building, documenting or testing the code. License finding curations are used to fix incorrect scan results, for example if a wrong license was detected, or if a finding is a false positive.
```

Comment:
artifacts -> artifact

Reply:
One package can have multiple release artifact, changed it to "part of the distributed release artifact(s) for a package"